src/components/layout/CyVerseAppBar.js

```diff
@@ -211,6 +216,52 @@ const BagMenu = () => {
     return <Bag menuIconClass={classes.menuIcon} />;
 };
 
+const DrawerItem = (props) => {
```

Comment:
Splitting this out doesn't help the load size, but it did help my sanity a bit. I hope others agree this is nicer. I didn't do anything with the admin drawer yet, because it uses a different pattern to some degree.